docs/release_notes/0.0.62.md

```diff
@@ -5,6 +5,8 @@
 ## Bugfixes
 KM281 🐛 Move ArgoCD CRD to environment specific folder (#559)
 
+KM284 🐛 Fix base/deployment containing env specific attribute image
```

Comment:
Add `(#560)` ?

Let me know if you don't do it like this (because of the clumsy workflow of having to create a PR and then update release notes), and I'll stop nagging about it.

Reply:
I dont like it, and I'm unsure about the value of doing it. However if anyone else thinks its a nice to have I'll keep doing it (and keep accepting nags about it)

Reply:
The value is so when people read the release notes (https://github.com/oslokommune/okctl/releases), it's simple to click the link to the pull request. And it's how gitrelease plugin would have done it if we had let it do its thing (except it uses commits, not PRs), see https://github.com/oslokommune/aws-delete-stacks/releases for example.

If not you have to read through the git commit log to find the code that the release is talking about - and hope the author didn't forget to put KMxxx in the commit message.

I actually think we need a task to fix the gitreleaser do stuff automatically, like for https://github.com/oslokommune/aws-delete-stacks/releases.

Reply:
👍👍👍 I'll keep on adding them then